views/index.mustache

```diff
@@ -1,12 +1,43 @@
-<html>
-  <head>
-    <title>{{ title }}</title>
-    <link rel="stylesheet" href="/style/style.css">
-  </head>
-  <body>
-    <h1>{{ title }}</h1>
-    <p>Welcome to {{ title }}</p>
-    <button id="logout">Log Out</button>
-    <script type="text/javascript" src="/script/logout.js"></script>
-  </body>
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang = en>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HomePage</title>
+    <link href = "visual.css" rel = "stylesheet" type = "text/css">
```

Comment:
is visual.css a placeholder? it isn't yet a file in public/style.

Reply:
Yes, that should be style.css